src/Administration/Resources/app/administration/src/module/sw-settings-services/component/sw-settings-services-service-card/sw-settings-services-service-card.spec.js

```diff
@@ -0,0 +1,279 @@
+import { mount } from '@vue/test-utils';
+import SwStatus from 'src/app/component/utils/sw-status';
+import {
+    MtModalAction,
+    MtModalRoot,
+    MtModal,
+    MtModalTrigger,
+    MtPopover,
+    MtPopoverItem,
+    MtButton,
+} from '@shopware-ag/meteor-component-library';
+import SwSettingsServicesServiceCard from './index';
+import SwColorBadge from '../../../../app/component/utils/sw-color-badge';
+
+describe('src/module/sw-settings-services/component/sw-settings-services-service-card.ts', () => {
+    beforeAll(() => {
+        Shopware.Service().register('shopwareExtensionService', () => ({
+            activateExtension: jest.fn(),
+            deactivateExtension: jest.fn(),
+        }));
+
+        Object.defineProperty(window, 'location', {
+            configurable: true,
+            value: { reload: jest.fn() },
+        });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it.each([
+        [
+            true,
+            [],
+            'green',
+            'active',
+        ],
+        [
+            false,
+            [],
+            'red',
+            'inactive',
+        ],
+        [
+            false,
+            ['order:read'],
+            'red',
+            'inactive',
+        ],
+        [
+            true,
+            ['order:read'],
+            'orange',
+            'awaiting-permissions',
+        ],
+    ])('displays the service with the correct status', (active, requestedPrivileges, statusColor, statusText) => {
+        const card = mount(SwSettingsServicesServiceCard, {
+            props: {
+                service: {
+                    id: 'service-id',
+                    active: active,
+                    name: 'service-name',
+                    label: 'service-label',
+                    icon: 'service-icon',
+                    description: 'service-description',
+                    updated_at: new Date('2025-07-08T00:00:00Z'),
```

Comment:
it's a string not a Date

```suggestion
                    updated_at: '2025-07-08 11:21:44.819',
```

Reply:
Are you sure? 🤔 This is the definition:

```js
export type ServiceDescription = {
    id: string;
    active: boolean;
    name: string;
    label: string;
    icon: string;
    description: string;
    updated_at: Date;
    version: string;
    requested_privileges: string[];
    privileges: string[];
};
```

Reply:
Than my definition is wrong. How should the PHP Api send a JS Date object? We don't mutate them

Reply:
This is what is in the component:

![image](https://github.com/user-attachments/assets/b32954ba-d80f-47bc-85d1-6e74a23b6381)

Reply:
Aight, lemme fix it then! 😄